src/scripts/mentoring-data-migration.js

```diff
@@ -0,0 +1,995 @@
+			},
+			{
+				name: 'notification_templates',
+				columns: { organization_id: 'organization_id' },
+				updateColumns: ['tenant_code', 'organization_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'organization_extension',
+				columns: { organization_id: 'organization_id' },
+				updateColumns: ['tenant_code', 'organization_code'],
+				hasPartitionKey: true,
+				primaryKey: ['tenant_code', 'organization_id'],
+			},
+			{
+				name: 'question_sets',
+				columns: { organization_id: 'organization_id' },
+				updateColumns: ['tenant_code', 'organization_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'questions',
+				columns: { organization_id: 'organization_id' },
+				updateColumns: ['tenant_code', 'organization_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'report_queries',
+				columns: { organization_id: 'organization_id' },
+				updateColumns: ['tenant_code', 'organization_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'reports',
+				columns: { organization_id: 'organization_id' },
+				updateColumns: ['tenant_code', 'organization_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'role_extensions',
+				columns: { organization_id: 'organization_id' },
+				updateColumns: ['tenant_code', 'organization_code'],
+				hasPartitionKey: true,
+			},
+		]
+
+		// Tables with only user_id - process second using user_extensions updated data
+		this.tablesWithUserId = [
+			{
+				name: 'user_extensions',
+				columns: { user_id: 'user_id' },
+				updateColumns: ['tenant_code', 'organization_code'],
+				hasPartitionKey: true,
+				primaryKey: ['tenant_code', 'user_id'],
+			},
+			{
+				name: 'sessions',
+				columns: { user_id: 'created_by' },
+				updateColumns: ['tenant_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'session_attendees',
+				columns: { session_id: 'session_id' },
+				updateColumns: ['tenant_code'],
+				hasPartitionKey: true,
+				useSessionLookup: true, // Special flag to indicate session-based lookup
+			},
+			{
+				name: 'feedbacks',
+				columns: { user_id: 'user_id' },
+				updateColumns: ['tenant_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'connection_requests',
+				columns: { user_id: 'created_by' },
+				updateColumns: ['tenant_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'connections',
+				columns: { user_id: 'created_by' },
+				updateColumns: ['tenant_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'entities',
+				columns: { user_id: 'created_by' },
+				updateColumns: ['tenant_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'issues',
+				columns: { user_id: 'created_by' },
+				updateColumns: ['tenant_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'post_session_details',
+				columns: { user_id: 'created_by' },
+				updateColumns: ['tenant_code'],
+				hasPartitionKey: true,
+			},
+			{
+				name: 'resources',
+				columns: { user_id: 'created_by' },
```

Comment:
@sumanvpacewisdom we should't set default tenant_code for this ,check this again with  other tables as well